change/@azure-msal-browser-2020-10-07-18-01-18-browser-add-broadcast.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Add event api to msal-browser (#2394)",
+  "packageName": "@azure/msal-browser",
+  "email": "joarroyo@microsoft.com",
+  "dependentChangeType": "patch",
```

Comment:
I would consider this a minor version, as it adds a new feature set to the library.

Reply:
Do you mean major version?

Reply:
Nope, minor.

Reply:
Ah, I was looking at the wrong line. Changed.

Reply:
I believe `dependentChangeType` denotes how versions for libraries that depend on this one should be bumped. @pkanher617 can confirm. Currently it's just samples that depend on browser, so it should be none for now?

Reply:
Whoops, I read the diff wrong. I think @pkanher617 wants this set to "none" for now.

Reply:
Okay. Updated.